python/python-core/advanced-queues/prioritize-your-queue.md

```diff
@@ -52,18 +52,18 @@ If we want to reverse the sorting order (greatest priority first), we would have
 
 ```python
 class Enki(object):
-    def __init__(self, priority):
-       self.priority = priority
-       return
-    def __lt__(self, other):
-       return self.priority > other.priority
+  def __init__(self, priority):
+    self.priority = priority
+    return
+  def __lt__(self, other):
+    return self.priority > other.priority
 
  q = queue.PriorityQueue()
  q.put(Enki(55))
  q.put(Enki(3))
  q.put(Enki(100))
  while not q.empty():
-     print(q.get().priority)
+  print(q.get().priority)
```

Comment:
This looks like a single white space on my phone?

Reply:
Yeah, you're right, it is one, changed it back to 2